crates/syntax/src/ast/token_ext.rs

```diff
@@ -331,6 +355,11 @@ impl ast::FloatNumber {
         }
         Some(&text[suffix_start..])
     }
+
+    pub fn value(&self) -> Option<FloatTypeWrapper> {
+        let float_value = self.text().parse::<f64>().ok()?;
+        Some(FloatTypeWrapper::new(float_value))
+    }
```

Comment:
Similarly this should just return the f64

Reply:
Made changes for the same